pkg/network/ebpf/c/tracer-maps.h

```diff
@@ -176,4 +176,15 @@ struct bpf_map_def SEC("maps/do_sendfile_args") do_sendfile_args = {
     .namespace = "",
 };
 
+// Used to store ip(6)_make_skb args to be used in the
+// corresponding kretprobes
+struct bpf_map_def SEC("maps/ip_make_skb_args") ip_make_skb_args = {
```

Comment:
Do we need separate maps for IPv4 and IPv6? They are separate kernel functions could be on different CPUs at the same time?

Reply:
I believe we do not share arg maps between functions in the rest of our code, so I'd prefer to be consistent.

Reply:
Actually, nevermind. We do for `inet_bind` and `inet6_bind`.